packages/shared/friends/sagas.ts

```diff
@@ -1251,6 +1251,19 @@ function* handleUpdateFriendship({ payload, meta }: UpdateFriendship) {
 
           // Send messsage to renderer via rpc
           yield apply(friendRequestModule, friendRequestModule.approveFriendRequest, [approveFriendRequest])
+
+          // If the friend request has a non-empty message body, it must be registered as part of the messages history of that user
+          if (messageBody) {
+            const chatMessage: ChatMessage = {
+              messageId: uuid(),
+              messageType: ChatMessageType.PRIVATE,
+              sender: getUserIdFromMatrix(userId),
+              recipient: getUserIdFromMatrix(ownId),
+              timestamp: Date.now(),
+              body: messageBody
+            }
+            store.dispatch(sendPrivateMessage(userId, chatMessage))
```

Comment:
Usage of `store` form within a saga will throw an exception
```suggestion
            yield put(sendPrivateMessage(userId, chatMessage))
```

Reply:
Thx! I keep forgetting about this. I'll revert the commit, so it will not longer apply.